[2024-05-26T08:32:00.872+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_dag.ensure_bucket_task scheduled__2024-05-26T08:30:00+00:00 [queued]>
[2024-05-26T08:32:00.875+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_dag.ensure_bucket_task scheduled__2024-05-26T08:30:00+00:00 [queued]>
[2024-05-26T08:32:00.875+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-26T08:32:00.881+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ensure_bucket_task> on 2024-05-26 08:30:00+00:00
[2024-05-26T08:32:00.887+0000] {standard_task_runner.py:60} INFO - Started process 266 to run task
[2024-05-26T08:32:00.889+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_dag', 'ensure_bucket_task', 'scheduled__2024-05-26T08:30:00+00:00', '--job-id', '2278', '--raw', '--subdir', 'DAGS_FOLDER/fetch_and_store_dag.py', '--cfg-path', '/tmp/tmp0tvwjs15']
[2024-05-26T08:32:00.890+0000] {standard_task_runner.py:88} INFO - Job 2278: Subtask ensure_bucket_task
[2024-05-26T08:32:00.897+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-05-26T08:32:00.912+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_and_store_dag.ensure_bucket_task scheduled__2024-05-26T08:30:00+00:00 [running]> on host d904a9488f34
[2024-05-26T08:32:00.944+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_dag' AIRFLOW_CTX_TASK_ID='ensure_bucket_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-26T08:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-26T08:30:00+00:00'
[2024-05-26T08:32:00.945+0000] {fetch_and_store_dag.py:29} INFO - Ensuring MinIO bucket exists...
[2024-05-26T08:32:00.948+0000] {fetch_and_store_dag.py:35} INFO - Bucket already exists: mediae
[2024-05-26T08:32:00.948+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-26T08:32:00.952+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_dag, task_id=ensure_bucket_task, execution_date=20240526T083000, start_date=20240526T083200, end_date=20240526T083200
[2024-05-26T08:32:00.980+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-26T08:32:00.994+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
